R/pdf_document.R

```diff
@@ -109,6 +109,7 @@ pdf_document <- function(toc = FALSE,
   # template path and assets
   if (identical(template, "default")) {
 
+    if (!pandoc_available()) stop("Pandoc not found")
```

Comment:
Yep I definitely understand it. As I said, it should be very rare for them to run into this problem. Anyway, I'll change the error message and make it more clear.